azure-pipelines/localization.yml

```diff
@@ -0,0 +1,22 @@
+trigger:
+- main
+pr: none
+
+pool:
+  vmImage: windows-latest
```

Comment:
This should use:

```
    pool:
      name: 'VSEngSS-MicroBuild2022-1ES'
```

like the signing one so that we get all the compliance stuff pushed to us; we were explicitly asked to not run things on the "retail" pools for anything that ships in an MS product recently. Something related to auditing all the software on the build machines or something like that being incompatible with the retail pools' attempt to "add all the dependent things" to the machines.